10 LoaderSpinner/readme.md

```diff
@@ -0,0 +1,294 @@
+# 09 Redux
+
+In this sample we will be using [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker) to display a loding spinner while loading on asynchronous calls.
+To do so we will take as an startup point sample _09 Redux_.
+
+Summary steps:
+
+- Install [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker) 
+- Create a common component for a spinner
+- Create a CSS to make the spinner looks good.
+- Add HOC react-promise-tracker to the component
+- Update actions to use [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker) by wraping asynchronous calls. 
+
+## Prerequisites
+
+Install [Node.js and npm](https://nodejs.org/en/) (v6.6.0) if they are not already
+installed on your computer.
+
+> Verify that you are running at least node v6.x.x and npm 3.x.x by running `node -v` and `npm -v`
+in a terminal/console window. Older versions may produce errors.
+
+## Steps to build it
+
+- Copy the content of the `09 Redux` folder to an empty folder for the sample.
+
+- Install the npm packages described in the `package.json` and verify that it works:
+
+ ```bash
+ $ npm install
+ ```
+- We also need to install [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker).
+
+```bash
+$ npm install react-promise-tracker --save
+```
+
```

Comment:
Add a note indicating that we don't need to install typings for this library since it already has the type definitions included in the library package

Reply:
done!